README.adoc

```diff
@@ -11,12 +11,14 @@ projects: []
 :toc:
 :icons: font
 :source-highlighter: prettify
-:project_id: draft-gs-template
+:project_id: gs-spring-cloud-gcp
 This guide walks you through the process of creating a Spring application.
 
 == What you'll build
 
-You'll build a Spring application.
+A https://spring.io/guides/gs/spring-boot/[Spring Boot application] that exchanges messages
+through Google Cloud Pub/Sub. It uses the Spring Integration Google Cloud Platform (GCP)
+Pub/Sub channel adapters to integrate with Google Cloud Pub/Sub.
 
 
 == What you'll need
```

Comment:
I don't really get the use of that launcher, besides being a proxy to the Pub/Sub section of the Cloud Console. If I'm not missing anything, I prefer linking directly to the Pub/Sub section.